test/elixir/test/view_pagination_test.exs

```diff
@@ -0,0 +1,189 @@
+defmodule ViewPaginationTest do
+  use CouchTestCase
+
+  @moduletag :view_pagination
+
+  @moduledoc """
+  Integration tests for pagination.
+  This is a port of the view_pagination.js test suite.
+  """
+
+  @tag :with_db
+  test "basic view pagination", context do
+    db_name = context[:db_name]
+
+    docs = make_docs(0..99)
+    bulk_save(db_name, docs)
+
+    query_function = "function(doc) { emit(doc.integer, null); }"
+
+    0..99
+    |> Enum.filter(fn number -> rem(number, 10) === 0 end)
```

Comment:
This stepped range calculation looks a little bit tricky but i don't seen a more direct way to do it.

Reply:
In my opinion the only other way to do this is declaring directly the list like `[0, 10, 20, 30]` 😅